pkg/deps/config/suppliers.go

```diff
@@ -138,17 +138,17 @@ func NewSupplyQueryClientContextFn(pocketQueryNodeURL string) SupplierFn {
 // NewSupplyTxClientContextFn returns a function with constructs a ClientContext
 // instance with the given cmd and returns a new depinject.Config which is
 // supplied with the given deps and the new ClientContext.
-func NewSupplyTxClientContextFn(pocketTxNodeURL string) SupplierFn {
+func NewSupplyTxClientContextFn(txNodeGRPCURL *url.URL) SupplierFn {
 	return func(_ context.Context,
 		deps depinject.Config,
 		cmd *cobra.Command,
 	) (depinject.Config, error) {
 		// Temporarily store the flag's current value
-		tmp := cosmosflags.FlagNode
+		tmpGRPC := cosmosflags.FlagGRPC
```

Comment:
`cosmosflags.FlagGRPC` is not a  value, it's the string name  of the flag: `node-grpc`. I think this should be:

```suggestion
		// TODO_TECHDEBT(#223): retrieve value from viper instead, once integrated.
		tmpGRPC := cmd.Flags().GetString(cosmosflags.FlagGRPC)
```